pkg/core/redis.go

```diff
@@ -36,6 +37,9 @@ type RedisCommand struct {
 	FlushConfig bool   `json:"flushConfig,omitempty"`
 	RedisPath   string `json:"redisPath,omitempty"`
 	RequestNum  int    `json:"requestNum,omitempty"`
+	Key         string `json:"key,omitempty"`
```

Comment:
seems that the `key` is useless

Reply:
I should add a "`key` is `$1`" judgement, only when the `key` is `$1` to delete all the keys, otherwire only delete the `key`.

Reply:
OK, I think it's better to update the description of `key` in flag